verticapy/tests/vModel/test_xgb_classifier.py

```diff
@@ -182,15 +182,28 @@ def test_drop(self):
         )
         assert current_cursor().fetchone() is None
 
-    @pytest.mark.skip(reason="not yet available.")
+    @pytest.mark.skip(reason="needs Vertica 12.0.3")
```

Comment:
Please make skip conditional so that it will skip the test if Vertica version is smaller than 12.0.3.

Reply:
We will remove the conditions when we will get the version 12.0.3 in GitHub. I think that passing the test is enough. Otherwise, we will have to review all the tests as many will pass only for specific version and we did not put any conditions on them which will take a lot of time (looking for each function which version).